contracts/DCNTCrescendo.sol

```diff
@@ -43,21 +43,23 @@ contract DCNTCrescendo is
 
   // Token uri
   string private _uri;
+  string private _contractURI;
 
-  uint256 private step1;
-  uint256 private step2;
-  uint256 private hitch;
-  uint256 private takeRateBPS;
+  uint256 public step1;
+  uint256 public step2;
+  uint256 public hitch;
+  uint256 public takeRateBPS;
   uint256 public royaltyBPS;
 
   // id to supply
   mapping(uint256 => uint256) private _totalSupply;
   // id to current price
   mapping(uint256 => uint256) private _currentPrice;
 
-  uint256 private totalWithdrawn = 0;
+  uint256 public totalWithdrawn = 0;
 
-  bool public saleIsActive = false;
+  uint256 public saleStart;
```

Comment:
I've seen Zora use `uint64` for times. Not sure if this is overoptimization but wanted to bring it up: https://github.com/ourzora/zora-drops-contracts/blob/main/src/interfaces/IERC721Drop.sol#L129

Reply:
also, do we want to include `saleEnd`? or are we assuming creators will always want infinite minting time for Crescendo?